src/Build/Definition/ProjectCollection.cs

```diff
@@ -432,7 +432,7 @@ public static ProjectCollection GlobalProjectCollection
         /// This is the Windows file version (specifically the value of the FileVersion
         /// resource), not necessarily the assembly version.
         /// If you want the assembly version, use Constants.AssemblyVersion.
-        /// This is not the <see cref="P:Microsoft.Build.BuildEngine.ToolsetCollection.ToolsVersions*">ToolsetCollection.ToolsVersions</see>.
+        /// This is not a <see cref="Toolset"/>.
```

Comment:
@Forgind I had issues with `ToolsetCollection.ToolsVersions` since it was importing a deprecated project. So I tracked down `Microsoft.Build.Evaluation.Toolset` which seems to be the 'new' version?